client/verta/tests/unit_tests/conftest.py

```diff
@@ -27,3 +28,9 @@ def mock_conn() -> Connection:
 def mock_config() -> Configuration:
     """ Return a mocked object of the _internal_utils._utils.Configuration class for use in tests """
     return Configuration(use_git=False, debug=False)
+
+
+@pytest.fixture
+def mocked_responses():
+    with responses.RequestsMock() as rsps:
+        yield rsps
```

Comment:
Just moved this here to `contest.py` since it can now be reused but the new tests.